### PR TITLE
Add gubernator links in bootstrap

### PR DESCRIPTION
**Pull request by fejta**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1074)

### CODE REVIEW

**Comment by rmmh** [comment]
I'm not seeing the links in the jenkins go test-- shouldn't it be there?

**Comment by fejta** [comment]
This hasn't been checked in yet

File: jenkins/bootstrap_test.py [hunk 1]
```diff
@@ -253,6 +253,22 @@ def upload_text(self, *args, **kwargs):
...
+
+    def testNonGS(self):
+        uri = 'hello/world'
+        self.assertEquals('hello', bootstrap.gubernator_uri(self.create_path(uri)))
+
+    def testMultipleGs(self):
+        uri = 'gs://hello/gs://there'
+        self.assertEquals(
+            bootstrap.GUBERNATOR + '/hello/gs:',
+            bootstrap.gubernator_uri(self.create_path(uri)))
```
Comment (rmmh) [inline jenkins/bootstrap_test.py]: no normal happy case?

Comment (fejta) [inline jenkins/bootstrap_test.py]: Done

**Pull request merged**